### PR TITLE
New charts with outliercounts

### PR DESCRIPTION
**Pull request by snim2**
- New script `mark_outliers_in_json` to add outliers to Krun results files
- New script `outlier_summaries` to generate LaTeX documents with a summary of outliers per benchmark, VM and language variant
- Changes to `plot_krun_data` to allow outliers to be annotated in plots, with the new `--with_outliers` switch

### CODE REVIEW

File: mark_outliers_in_json.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+
+def find_outliers(data, window_size):
+    outliers = list()
+    for index, datum in enumerate(data):
+        l_slice, r_slice = _clamp_window_size(index, len(data), window_size)
+        window = data[l_slice:r_slice]
+        mean = numpy.mean(window)
+        five_sigma = 5 * numpy.std(window)
+        if datum > (mean + five_sigma):
```
Comment (cfbolz) [inline mark_outliers_in_json.py]: should we also consider outliers that are exceptionally slow?

Comment (vext01) [inline mark_outliers_in_json.py]: I think we should consider both fast and slow spike iterations as outliers.

**Comment by cfbolz** [comment]
thanks

**Comment by snim2** [comment]
Thanks!

**Pull request merged**